packages/react-next/src/components/Pivot/Pivot.base.tsx

```diff
@@ -106,7 +106,7 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef(
       );
     };
 
-    const onKeyPress = (itemKey: string, ev: React.KeyboardEvent<HTMLElement>): void => {
+    const onKeyDown = (itemKey: string, ev: React.KeyboardEvent<HTMLElement>): void => {
```

Comment:
I wonder if this change, while not showing any unexpected things in our uses, could make some things stop working for partners.

Reply:
looks like keypress is being deprecated in web standard. i can't think of how this being a risky change.

Reply:
Also, this is listening to key events on an internal component inside Pivot. I don't know what a partner could be doing with it that this could break?

I'm going to resolve this, but let me know if you disagree @khmakoto

Reply:
I think those two are very valid reasons for doing this. I'm fine leaving this as resolved, thanks!